src/app.rs

```diff
@@ -12,9 +12,9 @@ pub fn build_app() -> Command<'static> {
         .color(clap_color_choice)
         .setting(AppSettings::DeriveDisplayOrder)
         .dont_collapse_args_in_usage(true)
-        .after_help(
-            "Note: `fd -h` prints a short and concise overview while `fd --help` gives all \
-                 details.",
+        .after_help("Bugs can be reported on Github: https://github.com/sharkdp/fd\n
```

Comment:
I think you might need a `\` at the end of this line

Reply:
This line was updated (simplified) in a subsequent push.